appengine/go11x/tasks/handler/task_handler_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+func TestIndexHandler(t *testing.T) {
+	tests := []struct {
+		route  string
+		status int
+		body   string
+	}{
+		{
+			route:  "/",
+			status: http.StatusOK,
+			body:   "Hello, World!",
+		},
+		{
+			route:  "/404",
+			status: http.StatusNotFound,
+			body:   "404 page not found\n",
+		},
+	}
+
+	for _, test := range tests {
+		req, err := http.NewRequest("GET", test.route, nil)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		rr := httptest.NewRecorder()
+		handler := http.HandlerFunc(indexHandler)
+		handler.ServeHTTP(rr, req)
+
+		if status := rr.Code; status != test.status {
+			t.Errorf(
```

Comment:
Remove line breaks in the middle of this function call (it's OK for some samples since it can make it easier to read when embedded).